### PR TITLE
fix(replication) gitlab replication task num not correct

### PR DESCRIPTION
**Pull request by bitsf**
fix gitlab replication task num not correct
close https://github.com/goharbor/harbor/issues/13272

### CODE REVIEW

File: src/replication/adapter/gitlab/adapter.go [hunk 1]
```diff
@@ -168,6 +169,11 @@ func (a *adapter) searchByPattern(pattern string) []*Project {
 				if len(substrings) < 2 {
 					continue
 				}
+				if _, ok := projectset[substrings[1]]; ok {
+					continue
+				} else {
```
Comment (steven-zou) [inline src/replication/adapter/gitlab/adapter.go:174]:
```suggestion
				} 
```

Comment (steven-zou) [reply src/replication/adapter/gitlab/adapter.go:174]: no else needed

**Review by steven-zou — COMMENTED** [review]
Commented

**Review by ywk253100 — APPROVED** [review]

**Review by steven-zou — APPROVED** [review]
LGTM

**Pull request merged**